extensions/amp-story/0.1/amp-story-page.js

```diff
@@ -273,43 +202,15 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   hasAudio_() {
-    return this.pageElements_.some(pageElement => pageElement.hasAudio());
-  }
-
-
-  /**
-   * @return {boolean} true, if the page is completely loaded; false otherwise.
-   * @public
-   */
-  calculateLoadStatus() {
-    if (this.isLoaded_ || this.pageElements_.length == 0) {
-      return true;
-    }
-
-    let isPageLoaded = true;
-    let canPageBeShown = false;
-
-    this.pageElements_.forEach(pageElement => {
-      pageElement.updateState();
-
-      if (isPageLoaded) {
-        isPageLoaded = pageElement.isLoaded || pageElement.hasFailed;
+    return Array.prototype.some.call(this.getAllMedia_(), mediaEl => {
+      if (!(mediaEl instanceof HTMLMediaElement)) {
+        return false;
       }
 
-      if (!canPageBeShown) {
-        canPageBeShown = pageElement.canBeShown;
-      }
+      return mediaEl.mozHasAudio ||
```

Comment:
This is not the right definition!  No one should look at this as a good example of how to detect whether a media element has audio, because it's not right!  This causes #11857, but this was the previous definition; I'm just porting it over from `page-elements.js` as part of the refactor in this PR.

Reply:
got it.